packages/desktop-client/src/components/budget/BudgetTotals.js

```diff
@@ -21,7 +21,7 @@ const BudgetTotals = memo(function BudgetTotals({
     <View
       data-testid="budget-totals"
       style={{
-        backgroundColor: 'white',
+        backgroundColor: theme.tableHeaderBackground,
```

Comment:
The light mode doesn't use a different tableheaderbackground here. We should follow this.
```suggestion
        backgroundColor: theme.tableBackground,
```